airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -284,13 +284,7 @@ def cancel_running_jobs(self, application_id: str, waiter_config: dict = {}):
                 )
                 for job_id in job_ids:
                     self.conn.cancel_job_run(applicationId=application_id, jobRunId=job_id)
-        if count > 0:
-            self.log.info("now waiting for the %s cancelled job(s) to terminate", count)
-            self.get_waiter("no_job_running").wait(
-                applicationId=application_id,
-                states=list(self.JOB_INTERMEDIATE_STATES.union({"CANCELLING"})),
-                WaiterConfig=waiter_config,
-            )
+        return count
```

Comment:
isn't that a breaking change ?
Also, if you were to go with it, you could remove the waiter_config param.